src/dashboard/cd-dashboard-stats.vue

```diff
@@ -1,28 +1,31 @@
 <template>
   <div class="column">
     <div class="cd-dashboard-stats">
-      <h1 class="cd-dashboard-stats__header">{{ $t('Dojo Stats') }}</h1>
-      <div class="cd-dashboard-stats__number">
-        <div class="cd-dashboard-stats__number-value"><span>{{ bookedChildren }}</span></div>
-        <div class="cd-dashboard-stats__description">{{ $t('kids attended your events') }}</div>
-      </div>
-      <div class="cd-dashboard-stats__pie">
-        <div class="cd-dashboard-stats__circle">
-          <svg viewBox="0 0 32 32">
-            <g v-for="gender in genderStats" >
-              <circle r="16" cx="16" cy="16" :style="{ strokeDashoffset: -gender.prevValue, strokeDasharray: `${gender.perc} 100`}"
-                :class="['cd-dashboard-stats__circle--' + gender.name]">
-              </circle>
-            </g>
-          </svg>
-          <div class="cd-dashboard-stats__circle-legends">
-            <div v-for="gender in genderStats" class="cd-dashboard-stats__circle-legend">
-              <div class="cd-dashboard-stats__circle-legend-color" :class="['cd-dashboard-stats__circle--' + gender.name]"></div>
-              <div stroke="#51c5cf" stroke-width="2px" text-anchor="middle" alignment-baseline="middle"> {{ gender.name }}</div>
+      <h3 class="cd-dashboard-stats__header">{{ $t('Dojo Stats') }}</h3>
```

Comment:
This should be h2

Reply:
i changed it in this merged pr https://github.com/heyitsmaimai/cp-zen-frontend/blob/6637c610a106774162a07b2524ae6481b5ddb6ec/src/dashboard/cd-dashboard-children.vue

Reply:
Then the children name should be h3, not h4 :)

Reply:
no no h4, conor did a style review earlier with me

Reply:
so the new news and children have the correct stylings

Reply:
er we could set variables for each font size?

Reply:
To explain a bit further because you seem a bit lost : `<h[n]>` tags are supposed to be following each another and be a logical evolution. `<h[n]>` are not about styling, they are about structuring a page. If you're having a word document with a `h1` default styling and then a h3 because you prefer how it looks, don't be surprised that your table of content is fucked up : the `<h1>` tag does not correspond to a styling, but to a hierarchy. The styling is "our" fault.
That's why I said that harmonizing styles is its own issue and should not be put on top the dashboard as if it was a lightweight task. (cc @conoramurphy, @DanielBrierton ). Using `<h[n]>` tag when it's not a title is not a solution, nor is repeating styles from the FTB by copy-pasting, nor is overwriting `<h[n]>` styling in this page. That's a work of review and replace over all existing pages to >harmonize< style and tag usages.

Reply:
On the usage of `<h[n]>` tags for semantics vs. styling:

![no-strong-feelings](https://user-images.githubusercontent.com/181418/45310688-e7f46500-b51e-11e8-9822-4ce87f7419c6.gif)

Reply:
Well, it fucks up screen reader, as a wild guess? HTML is a format markup language, not a styling one.